definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "homer_tag_directory.sh"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "chrisamiller/homer"
+    - class: ResourceRequirement
+      ramMin: 32000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'homer_tag_directory.sh'
+        entry: |
+		set -o pipefail
+		set -o errexit
+
+		name=$1
+		bam=$2
+		reference_fasta=$3  #assumes that there's a .genome file next to the reference .fa
+		#run in chrisamiller/homer container
```

Comment:
We should probably check for the presence of that genome file here and throw an informative error if it's missing.  (or possibly even require it as an input, like we do in some other pipelines)